src/main/java/spoon/support/reflect/declaration/CtTypeImpl.java

```diff
@@ -891,9 +892,16 @@ public String getQualifiedName() {
 	public Set<CtMethod<?>> getAllMethods() {
 		final Set<String> distinctSignatures = new HashSet<>();
 		final Set<CtMethod<?>> l = new SignatureBasedSortedSet<>();
```

Comment:
Actually I may have found an issue which prevent me from removing the set, see #1407. I propose to keep it as it is (it works well now) and to remove it later if needed following the change made for #1407